### PR TITLE
DBot Truth Bombs Content Pack

### PR DESCRIPTION
**Pull request by asafshen**
## Status
- [x] In Progress

## Description
DBot Facts Pack

## Screenshots
TBD

### CODE REVIEW

File: Packs/DBotFacts/Scripts/FactsAboutYou/FactsAboutYou.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+name: FactAboutYou
+runas: DBotWeakRole
+runonce: false
+script: ''
+scripttarget: 0
+subtype: python3
+tags:
+- Utilities
+type: python
+fromversion: 6.0.0
```
Comment (yaakovi) [inline Packs/DBotFacts/Scripts/FactsAboutYou/FactsAboutYou.yml:30]: why is the script from 6.0 and integration is from 5.0?

**Review by yaakovi — APPROVED** [review]
images 1, 2, 5, and 6 don't have a transparent background, is that ok?

@graphicool FYI

**Comment by asafshen** [comment]
thanks for the review!

> images 1, 2, 5, and 6 don't have a transparent background, is that ok?
 
reg the images, I took them from the product, in order to increase variety

**Comment by graphicool** [comment]
> thanks for the review!
> 
> > images 1, 2, 5, and 6 don't have a transparent background, is that ok?
> 
> reg the images, I took them from the product, in order to increase variety

i want us to use only the memes, and i dont mind adding variety and doing more. hit me with ideas.

**Pull request merged**